experiments/AMIP/moist_mpi_earth/push_pull.jl

```diff
@@ -13,17 +13,16 @@ function atmos_push!(cs)
 end
 
 """
-   land_pull!(cs)
+   land_pull!(FT, cs)
 
 Updates the land_sim cache state in place with the current values of F_A, F_R, F_E, P_liq, and ρ_sfc.
 The surface air density is computed using the atmospheric state at the first level and making ideal gas
 and hydrostatic balance assumptions. The land model does not compute the surface air density so this is
 a reasonable stand-in.
 """
-function land_pull!(cs)
+function land_pull!(FT, cs)
```

Comment:
```suggestion
function land_pull!(cs)
```
We can keep this interface the same and just replace `FT = cs.FT` with `FT = float_type(cs)` below